src/components/Form/ExitFormDialog.tsx

```diff
@@ -19,6 +18,12 @@ const useStyles = makeStyles(
     buttonsContainer: {
       display: "flex",
       justifyContent: "flex-end"
+    },
+    dialogContent: {
+      "@media (min-width: 800px)": {
+        minWidth: 500
```

Comment:
wouldn't `<Dialog maxWidth="sm" fullWidth />` suffice?

Reply:
Technically yes, but it doesn't exactly match designs.
Current:
![image](https://user-images.githubusercontent.com/13994677/157860572-0719e39c-78e0-412d-ba64-41a56f877590.png)

fullWidth + "sm"
![image](https://user-images.githubusercontent.com/13994677/157860479-9ecbf8fd-b11d-490a-b382-c4cdd9a12418.png)